docs/internal/api-spec.swagger.yml

```diff
@@ -419,6 +406,9 @@ definitions:
       position:
         type: integer
         description: The index of the output among the transaction's outputs.
+      output_id:
```

Comment:
I agree with [@dominic's comment](https://github.com/chain/chain/pull/421#pullrequestreview-18059111) that in this context, this field should be simply `id` rather than `output_id`.